lib/ezimage/classes/ezimagemanager.php

```diff
@@ -1059,6 +1084,17 @@ function convert( $sourceMimeData, &$destinationMimeData, $aliasName = false, $p
                 {
                     eZMimeType::changeMIMEType( $destinationMimeData, $alias['mime_type'] );
                 }
+                $qualityOverride = $alias['quality'];
+                if ( $qualityOverride )
+                {
+                    foreach ( $qualityOverride as $qualityOverrideValue )
+                    {
+                        $elements = explode( ';', $qualityOverrideValue );
+                        $mimeType = $elements[0];
```

Comment:
Maybe do a basic check if `$elements[0]` and `$elements[1]` exist before using them?

Reply:
I was copying readMIMETypeQualitySettingFromINI() on line 559, so it's consistent to expect the user to follow the right format. Could harden this in a follow-up PR though.